specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1578 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager endpoint."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "heatMapType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "description": "The type of HeatMap for the Traffic Manager profile."
+          },
+          {
+            "name": "topLeft",
+            "in": "query",
+            "required": false,
+            "allowEmptyValue": true,
+            "description": "The top left latitude,longitude pair of the rectangular viewport to query for.",
+            "type": "array",
+            "minItems": 2,
+            "maxItems": 2,
+            "collectionFormat": "csv",
+            "items": {
+              "type": "number",
+              "format": "double"
+            }
+          },
+          {
+            "name": "botRight",
+            "in": "query",
+            "required": false,
+            "allowEmptyValue": true,
+            "description": "The bottom right latitude,longitude pair of the rectangular viewport to query for.",
+            "type": "array",
+            "minItems": 2,
+            "maxItems": 2,
+            "collectionFormat": "csv",
+            "items": {
+              "type": "number",
+              "format": "double"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Traffic Manager heatmap.",
+            "schema": {
+              "$ref": "#/definitions/HeatMapModel"
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "HeatMap-GET": {
+            "$ref": "./examples/HeatMap-GET.json"
+          },
+          "HeatMap-GET-With-TopLeft-BotRight": {
+            "$ref": "./examples/HeatMap-GET-With-TopLeft-BotRight.json"
+          },
+          "HeatMap-GET-With-Null-Values": {
+            "$ref": "./examples/HeatMap-GET-With-Null-Values.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/trafficManagerUserMetricsKeys": {
```

Comment:
Is it at subscription scope or should it live at resource group scope?

Reply:
It is at subscription scope.